2020_RTSS_cpc_model_based_algorithm/src/main.rs

```diff
@@ -20,13 +22,34 @@ struct ArgParser {
     number_of_cores: usize,
     #[clap(short = 'o', long = "output_dir_path", default_value = "../outputs")]
     output_dir_path: String,
+    #[clap(short = 'p', long = "period_factor", default_value = "1.0")]
```

Comment:
The following is clearer.
In addition, please add a sentence explaining the arguments, if possible.

```suggestion
    #[clap(short = 'p', long = "ratio_deadline_to_period", default_value = "1.0")]
```

Reply:
![Uploading image.png…]()

Reply:
Added help message.